src/lib.rs

```diff
@@ -377,7 +379,7 @@ impl<K: KeyExtractor> GovernorConfigBuilder<K> {
 /// Configuration for the Governor middleware.
 pub struct GovernorConfig<K: KeyExtractor> {
     key_extractor: K,
-    limiter: Arc<RateLimiter<K::Key, DefaultKeyedStateStore<K::Key>, DefaultClock>>,
+    limiter: Arc<RateLimiter<K::Key, DefaultKeyedStateStore<K::Key>, DefaultClock, StateInformationMiddleware>>,
```

Comment:
It would be great if you could disable the `StateInformationMiddleware` by default. Just like the `KeyExtractor` part is optional and uses peer IPs by default. The `StateInformationMiddleware` simply comes with a performance penalty and some users probably don't want to enable it.

Reply:
Oh, I never thought of this, I'll work on it